orbitx/flight_gui.py

```diff
@@ -751,102 +766,5 @@ def rate(self, framerate: int) -> None :
         """Alias for vpython.rate(framerate). Basically sleeps 1/framerate"""
         self._vpython.rate(framerate)
     # end of rate
+# end of class FlightGui
 
-
-VPYTHON_CSS = """<style>
```

Comment:
thanks for putting these variables in their own file, it makes this file much tidier